sklearn/feature_extraction/_hashing_fast.pyx

```diff
@@ -35,7 +35,7 @@ def transform(raw_X, Py_ssize_t n_features, dtype,
     # and values arrays ourselves. Use a Py_ssize_t capacity for safety.
     cdef Py_ssize_t capacity = 8192     # arbitrary
     cdef cnp.int64_t size = 0
-    cdef cnp.ndarray values = np.empty(capacity, dtype=dtype)
+    cdef cnp.dtype values = np.empty(capacity, dtype=dtype)
```

Comment:
If you can type `raw_X` as a memoryview, then you should be able to access the type to type `values` here. :)

Reply:
I think that is this case we do not need to change the `cnp.ndarray` as it is explained [here](https://github.com/scikit-learn/scikit-learn/pull/25574). What do you think?
should this pr be closed and marked _hashing_fast.pyx as resolved in the initial issue?

Reply:
OK, let's close this for now.